scan_filter_chain:
- name: nan_to_inf_filter
  type: segbot_sensors/NanToInfFilter
- name: angle
  type: segbot_sensors/AngleRangeFilter
  params:
    angle_min: -2.09234976768
    angle_max:  2.09234976768
- name: shadows
  type: laser_filters/ScanShadowsFilter
  params:
    min_angle: 10
    max_angle: 170
    neighbors: 1
    window: 1
- name: footprint_filter
  type: segbot_sensors/SegbotFootprintFilter
  params:
    footprint_frame: base_footprint
    footprint: [0.2,0.3,0.2,-0.3,-0.35,-0.25,-0.35,0.25]

